### PR TITLE
Karam dosa

### PR DESCRIPTION
**Pull request by vikram1503**

### CODE REVIEW

**Comment by vikrampractice** [comment]
why u added oil to the karam dosa?u should have added ghee

**Comment by vikrampractice** [comment]
> why u added oil to the karam dosa?u should have added ghee

for cost purpose

**Comment by vikram1503** [comment]
> why u added oil to the karam dosa?u should have added ghee

to cut down the costs

File: recipes/dosa.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+dosa
+karam
+oil
```
Comment (vikrampractice) [inline recipes/dosa.txt:3]: why oil is added

**Review by vikrampractice — APPROVED** [review]
everything is ok

**Pull request merged**